alphastats/gui/pages/05_LLM.py

```diff
@@ -88,9 +88,9 @@ def llm_config():
     st.info("Create a Volcano plot first using the 'Analysis' page.")
     st.stop()
 
-volcano_plot, parameter_dict = st.session_state[StateKeys.LLM_INPUT]
+volcano_plot, plot_parameters = st.session_state[StateKeys.LLM_INPUT]
```

Comment:
wouldn't this be analysis parameters, if we ever generalize to all analyses being interpretable by LLM?

Reply:
yes but for now it's tied to the volcano, that's we I chose to name it like so